Makefile

```diff
@@ -14,7 +14,7 @@ install-gen-deps: ## Install dependencies for code generation
 	# out code dependencies; so a go mod tidy will remove them again. `go
 	# install` seems to install the last tagged version and we want to install
 	# master.
-	@(cd $(TEMPDIR) && GO111MODULE=on go get github.com/mna/@master)
+	@(cd $(TEMPDIR) && GO111MODULE=on go get github.com/mna/pigeon@master)
 	@(cd $(TEMPDIR) && GO111MODULE=on go get github.com/alvaroloes/enumer@master)
```

Comment:
Hello there, as per https://github.com/golang/go/issues/25922, another thing we _could_ do here to not have to run these steps is having a `tools.go` file that could look like so:

```go
// +build tools

package tools

import (
  _ "github.com/mna/pigeon"
  _ "github.com/alvaroloes/enumer"
)
```

This way these tools will be checked in the code for us and versioned by go.
Another cool thing we can do is change the go generate calls with something like:

```
//go:generate go run golang.org/x/tools/cmd/stringer@v1.11.0 -type=Pill
```

Reply:
looks like the latter suggestion doesn't actually work; it was an idea. I'm tinkering with trying to get a to tools file working now

Reply:
I spent 15 minutes tinkering with this and couldn't get it working so I reverted; if we end up having to manage more dependencies in the future, or if golang ends up actually implementing something along the lines of 

>//go:generate go run golang.org/x/tools/cmd/stringer@v1.11.0 -type=Pill 

(which doesn't currently work, it was just an idea in that thread), then we can revisit it.

Reply:
Ah right bummer, sorry ! If the projects had more tags the header version could have worked, but it's not the case.